ape_solidity/_utils.py

```diff
@@ -4,17 +4,18 @@
 import subprocess
 from enum import Enum
 from pathlib import Path
-from typing import Dict, List, Optional, Set, Union
+from typing import Dict, List, Set, Union
 
 from ape.exceptions import CompilerError
 from ape.logging import logger
+from packaging.specifiers import SpecifierSet
 from packaging.version import InvalidVersion
+from packaging.version import Version
 from packaging.version import Version as _Version
 from pydantic import BaseModel, validator
-from semantic_version import NpmSpec, Version  # type: ignore
 from solcx.exceptions import SolcError  # type: ignore
 from solcx.install import get_executable  # type: ignore
-from solcx.wrapper import VERSION_REGEX  # type: ignore
+from solcx.wrapper import VERSION_REGEX, get_solc_version  # type: ignore
```

Comment:
this PR will publish types so we'd be able to delete these type ignores: https://github.com/ApeWorX/py-solc-x/pull/162